scripts/helpers.sh

```diff
@@ -90,9 +90,14 @@ get_interfaces()
     local interfaces=$(get_tmux_option @net_speed_interfaces "")
 
     if [[ -z "$interfaces" ]] ; then
-        for interface in /sys/class/net/*; do
-            interfaces+=$(echo $(basename $interface) " ");
-        done
+        if [[ -n is_osx ]]
```

Comment:
This if statement is always true because `-n` is used to determine if the length of a string is zero.

The correct statement should be `if is_osx`.

I did a quick fix https://github.com/sainnhe/tmux-net-speed/commit/2e94ad2397441e03d5f9d0a4e91094447f66d2f8, you can merge my commit if you would like to.

Reply:
I tried `if [[ is_osx ]]` and it works well!